namespace java genapi
namespace cpp config

service ConfigApi {
	void addNode()
}
